Log std tests fail due to race detection in Go v1.19.1

While running the unit tests with `make test` using Go v1.19.1, the `log/std` package-related tests fail, due to race detection.

Stack trace of failing tests:
```
--- FAIL: TestLogMetrics (0.00s)
    --- FAIL: TestLogMetrics/error (0.00s)
        testing.go:1319: race detected during execution of test
    --- FAIL: TestLogMetrics/panic (0.00s)
        testing.go:1319: race detected during execution of test
    --- FAIL: TestLogMetrics/warn (0.00s)
        testing.go:1319: race detected during execution of test
--- FAIL: TestLogger_shouldLog (0.00s)
    testing.go:1319: race detected during execution of test
FAIL
```

> `patron` could notice such issues in the future, by adding matrix tests for supported Go versions.

--- a/log/std/logger_test.go
+++ b/log/std/logger_test.go
@@ -36,7 +36,6 @@ new lines and	tabs`)
 }
 
 func TestLogMetrics(t *testing.T) {
-	t.Parallel()
 	log.ResetLogCounter()
 	var b bytes.Buffer
 	l := New(&b, log.InfoLevel, nil)
@@ -54,7 +53,6 @@ func TestLogMetrics(t *testing.T) {
 	for name, tt := range tests {
 		tt := tt
 		t.Run(name, func(t *testing.T) {
-			t.Parallel()
 			assert.Equal(t, 0.0, testutil.ToFloat64(log.LevelCount(string(tt.lvl))))
 			if tt.lvl == log.PanicLevel {
 				assert.Panics(t, func() { tt.logfunc(name) })

